packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -864,12 +878,18 @@ test('interest on multiple vaults', async t => {
   }
   await waitForPromisesToSettle();
 
+  const assetUpdate = await E(assetNotifier).getUpdateSince();
   const aliceUpdate = await E(aliceNotifier).getUpdateSince();
   const bobUpdate = await E(bobNotifier).getUpdateSince();
+
   // 160 is initial fee. interest is 3n/week. compounding is in the noise.
   const bobAddedDebt = 160n + 3n;
   t.deepEqual(
-    bobUpdate.value.debt,
+    calculateCurrentDebt(
```

Comment:
I think this verifies that `calculateCurrentDebt` was used in calculating Bob's debt. We should also have unit tests showing that `calculateCurrentDebt` produces the correct number.